map/src/data/markers.ts

```diff
@@ -27,6 +27,28 @@ export interface MarkerInfo {
 }
 
 const LOCATIONS = {
+  DE: {
+    BERLIN: {
+      locationDescription: 'Berlin, Germany',
+      lat: 52.511528,
+      lng: 13.406853,
+      serviceRadius: 24000,
+    },
+  },
+  GB: {
+    NEWCASTLE_TYNE: {
+      locationDescription: 'Newcastle upon Tyne, UK',
+      lat: 54.975601,
+      lng: -1.613076,
+      serviceRadius: 9000,
+    },
+    UK: {
```

Comment:
The UK encompasses GB (it's the United Kingdom of Great Britain and Northern Island), so it probably makes sense to replace `GB` with `UK` here, and then `UK` could become `ALL` perhaps?

Reply:
Lol yes. I legit have no idea what's correct. It's been so long since I worked on anything touching that part of the world that I don't remember what it's supposed to be called where 😛.

My recommendation is to have you rename it to whatever is most appropriate.